update blog to nextra 4 (#1783)
the-guild-org/website: readme.md

## Build and Run the Website (Production)

Run `pnpm build` to build a static version of the website in production mode.

## Contributing to Our Blog

We use [MDX](https://mdxjs.com) (Markdown with JSX) for our blog posts. To add a new blog post:

> - Create a new MDX file in `website/content` directory with the same template as other blog posts

Author: ```suggestion
- Create a new MDX file in `website/app/blog/(current-year)` directory with the same template as other blog posts
```

- Update the meta object with all the information regarding the blog post including title,
  thumbnails, author name, etc.

> - Any assets related to the blog can be placed in `website/public/blog-assets` directory within its

Author: ```suggestion
- Any assets related to the blog can be placed directly in your blog folder
```
